packages/rke2-cilium/generated-changes/patch/values.yaml.patch

```diff
@@ -1,70 +1,69 @@
 --- charts-original/values.yaml
 +++ charts/values.yaml
-@@ -145,12 +145,10 @@
- # -- Agent container image.
- image:
+@@ -152,12 +152,10 @@
+   # type: [null, string]
+   # @schema
    override: ~
 -  repository: "quay.io/cilium/cilium"
 +  repository: "rancher/mirrored-cilium-cilium"
-   tag: "v1.15.7"
+   tag: "v1.16.0"
    pullPolicy: "IfNotPresent"
 -  # cilium-digest
--  digest: "sha256:2e432bf6879feb8b891c497d6fd784b13e53456017d2b8e4ea734145f0282ef0"
+-  digest: "sha256:46ffa4ef3cf6d8885dcc4af5963b0683f7d59daa90d49ed9fb68d3b1627fe058"
 -  useDigest: true
 +  useDigest: false
- 
  # -- Affinity for cilium-agent.
  affinity:
-@@ -561,7 +559,9 @@
+   podAntiAffinity:
+@@ -581,7 +579,8 @@
    #  - flannel
    #  - generic-veth
    #  - portmap
 -  chainingMode: ~
-+
-+  # Otherwise rke2 hostPort does not work! Used for nginx
++# Otherwise rke2 hostPort does not work! Used for nginx
```

Comment:
nit: looks like whitespace changed on this comment? Not sure if that's important for the yaml structure?